protocol/x/clob/keeper/process_single_match.go

```diff
@@ -307,6 +309,7 @@ func (k Keeper) persistMatchedOrders(
 	makerFeePpm int32,
 	bigFillQuoteQuantums *big.Int,
 	insuranceFundDelta *big.Int,
+	affiliatesWhitelistMap map[string]uint32, // nolint: unused
```

Comment:
Why the `nolint`? Isn't `affiliatesWhitelistMap`used below? 

In general `nolint` in non-testing files is red flag

Reply:
this was a remnant from before the clob pr was merged. Should be removed